test/e2e/kubeproxy.go

```diff
@@ -225,13 +228,17 @@ func (config *KubeProxyTestConfig) dialFromContainer(protocol, containerIP, targ
 		targetPort,
 		tries)
 
-	By(fmt.Sprintf("Dialing from container. Running command:%s", cmd))
-	stdout := config.ssh(cmd)
-	var output map[string][]string
-	err := json.Unmarshal([]byte(stdout), &output)
-	Expect(err).NotTo(HaveOccurred(), fmt.Sprintf("Could not unmarshal curl response: %s", stdout))
-	hostNamesMap := array2map(output["responses"])
-	Expect(len(hostNamesMap)).To(BeNumerically("==", expectedCount), fmt.Sprintf("Response was:%v", output))
+	if providerIs(providersWithSSH...) {
```

Comment:
I think we should use kubectl exec into a net=host pod and run the shell commands to work around the no-ssh-available problem.
For purposes of this PR, I would just run the entire test  for providerWithSSH, otherwise skip the whole test.
We are skipping pretty much everything other than the LoadBalancer portion..